symplyphysics/laws/hydro/inner_pressure_of_fluid.py

```diff
@@ -0,0 +1,55 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.laws.hydro.dynamic_pressure_from_velocity import dynamic_pressure
+from symplyphysics.laws.hydro.hydrostatic_pressure_from_density_and_depth import hydrostatic_pressure
+
+# Description
+## Inner pressure of an ideal fluid is the sum of static, dynamic, and hydrostatic pressure at chosen point.
+
+# Law: P_inner = P_static + P_dynamic + P_hydrostatic
+## P_inner -- inner pressure
+## P_static -- static pressure
+## P_dynamic -- dynamic pressure
+## P_hydrostatic -- hydrostatic pressure
+
+# Condition: this definition applies to an ideal liquid, namely one that is
+## 1) nonviscous
+## 2) in steady (laminar) flow
+## 3) incompressible
+## 4) irrotational
+
+inner_pressure = Symbol("inner_pressure", units.pressure)
+static_pressure = Symbol("static_pressure", units.pressure)
+
+law = Eq(inner_pressure, static_pressure + dynamic_pressure + hydrostatic_pressure)
```

Comment:
Make dynamic_pressure and hydrostatic_pressure symbols. Do not import from laws.

Reply:
Wouldn't this somehow connect the laws between each other, or is it more of a convention to not do so?

Reply:
Okay, I see.